src/core/loaders/Loader.ts

```diff
@@ -200,6 +202,27 @@ export abstract class Loader extends Disposable {
     return str
   }
 
+  private treeNodeValueHasPluralizationKeys(value: Record<string, any>, locale: string) {
+    if (!value)
+      return false
+
+    return isObject(value)
+      && this.pluralizationKeysV4.some((key: string) => get(value, key) != null)
```

Comment:
You could simplify this to one line, and the `get` method wouldn't be needed:

```suggestion
    return value && isObject(value) && Object.keys(value).some((key) => this.pluralizationKeysV4.includes(key))
```